dot/build_spec_test.go

```diff
@@ -14,138 +14,3 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
 package dot
```

Comment:
should we remove this file?

Reply:
I kept this file to add unit tests to in another PR.